crates/conjure_core/src/rules/constant.rs

```diff
@@ -81,15 +82,50 @@ pub fn eval_constant(expr: &Expr) -> Option<Lit> {
                 return None;
             }
 
-            Some(Lit::Bool(a / b == c))
+            let a = a as f32;
+            let b = b as f32;
+            let div: i32 = (a / b).floor() as i32;
+            Some(Lit::Bool(div == c))
         }
         Expr::Bubble(_, a, b) => bin_op::<bool, bool>(|a, b| a && b, a, b).map(Lit::Bool),
 
         Expr::Reify(_, a, b) => bin_op::<bool, bool>(|a, b| a == b, a, b).map(Lit::Bool),
-        _ => {
-            warn!(%expr,"Unimplemented constant eval");
-            None
+        Expr::SumEq(_, exprs, a) => {
+            flat_op::<i32, bool>(|e, a| e.iter().sum::<i32>() == a, exprs, a).map(Lit::Bool)
+        }
+        Expr::ModuloEqUndefZero(_, a, b, c) => {
+            // modulo always rounds down
```

Comment:
I don't think this comment is correct. Modulo works in a way that is compatible with an always-rounding-down div.